fuseki/sample-ubd.trig

```diff
@@ -0,0 +1,280 @@
+base <https://cube-creator.lndo.site/>
+prefix schema: <http://schema.org/>
+prefix dcterms: <http://purl.org/dc/terms/>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix hydra: <http://www.w3.org/ns/hydra/core#>
+prefix csvw: <http://www.w3.org/ns/csvw#>
+prefix dtype: <http://www.linkedmodel.org/schema/dtype#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix void: <http://rdfs.org/ns/void#>
+prefix cc: <https://cube-creator.zazuko.com/vocab#>
+
+<ubd-example> a void:Dataset .
+
+<cube-project/ubd> void:inDataset <ubd-example> .
+graph <cube-project/ubd> {
+  <cube-project/ubd>
+    a cc:CubeProject, hydra:Resource ;
+    cc:dataset <foen/ubd/28/pm1> ;
+    cc:cube   </cube/cli-test> ;
```

Comment:
As per #107, this would become `cc:cubeGraph`

Reply:
We need to discuss this because I'm not sure I understood the idea